README.md

```diff
@@ -7,15 +7,62 @@ A set of load tests for OCM's clusters-service, based on vegeta.
 Compile using `make` and run as a simple binary:
 
 ```sh
-./ocm-api-load --token=<OCM_TOKEN>   \
-    --output-path="/path/to/output/" \
-    --gateway-url=<api-gateway-url>  \
-    --duration-in-min=1              \
+./ocm-api-load --config-file /path/to/config
 ```
 
-## CLI Parameters
+if your `config-file` is named `ocm-api-load.yaml` and it in the same
+path as your binary, it will be autodetected and you could run by just calling it:
 
-1. The `token` paramater is an auth token for OCM. One can obtain a token from the following [link](https://qaprodauth.cloud.redhat.com/openshift/token).
-2. The `gateway-url` parameter is a url to run the tests against, by default `http://localhost:8000` which is the default endpoint for the development environment.
-3. The `duration-in-min` parameter indicates how long should the test run in minutes.
-4. The `output-path` paramater indicates the path to output the results. For more information about the results format, please see [vegeta](https://github.com/tsenart/vegeta#report-command).
+```sh
+./ocm-api-load
+```
+
+## Config file
+
+### Minimal yaml config file
+
+```yaml
+---
+token: xxxXXXyyyYYYzzzZZZ
+gateway-url: https://api.integration.openshift.com
+```
+
+## Defaults
+
+- output-path: set to `./results`
+- duration-minutes: set to `1` minute
+
+### Full yaml config file
+
+```yaml
+---
+token: xxxXXXyyyYYYzzzZZZ           # Offline token for authentication.
+gateway-url: http://localhost:8000  # Gateway URL.
+client:
```

Comment:
I just noticed the OpenID authentication in here. What's that used for?

Reply:
For creating the sdk client - honestly these can be statically set in the code but we can keep them here as well.